src/main/cpp/subsystems/DriveSubsystem.cpp

```diff
@@ -64,6 +64,26 @@ DriveSubsystem::DriveSubsystem(Vision* vision)
       nt::NetworkTableInstance::GetDefault()
           .GetStructArrayTopic<frc::SwerveModuleState>("/SwerveStates")
           .Publish();
+
+  m_sysIdRoutine = std::make_unique<frc2::sysid::SysIdRoutine>(
+      frc2::sysid::Config{std::nullopt, std::nullopt, std::nullopt, nullptr},
+      frc2::sysid::Mechanism{
+          [this](units::volt_t driveVoltage) {
+            m_frontLeft.SetMotorVoltage(MotorType::DriveMotor, driveVoltage);
+            m_rearLeft.SetMotorVoltage(MotorType::DriveMotor, driveVoltage);
+            m_frontRight.SetMotorVoltage(MotorType::DriveMotor, driveVoltage);
+            m_rearRight.SetMotorVoltage(MotorType::DriveMotor, driveVoltage);
+          },
+          [this](frc::sysid::SysIdRoutineLog* log) {
+            log->Motor("drive-front-left")
```

Comment:
Magic string? Maybe a constant, perchance? Or maybe create a method that accepts a `frc::sysid::SysIdRoutineLog*`, name, and `MAXSwerveModule&` which gets called here instead to reduce copy-paste errors